numba_dppy/tests/test_debug.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env python
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import pytest
+
+import numba_dppy as dppy
+import dpctl
+
+from numba.core import types
+from numba_dppy import compiler
+from numba_dppy.tests.skip_tests import skip_test
```

Comment:
pylint shows following warnings:
```bash
numba_dppy/tests/test_debug.py:20:0: C0411: third party import "import dpctl" should be placed before "import numba_dppy as dppy" (wrong-import-order)
numba_dppy/tests/test_debug.py:22:0: C0411: third party import "from numba.core import types" should be placed before "import numba_dppy as dppy" (wrong-import-order)
```
I know that until CI will perform code style checks this kind of things will be common.
I will make this checks in CI soon. For now I can only recommend start pylint locally.